### PR TITLE
LVA on ASE sample with tutorial

### PR DESCRIPTION
**Pull request by julialieberman**
## Purpose

* This adds files necessary to run Live Video Analytics on the Azure Stack Edge. The tutorial is in the onenote as well as in the readme.md.

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  run through the tutorial! Onenote link is [here](https://microsoft.sharepoint-df.com/teams/AzureStackLearnings/_layouts/OneNote.aspx?id=%2Fteams%2FAzureStackLearnings%2FSiteAssets%2FAzureStack%20Learnings%20Notebook&wd=target%28Demo.one%7C8264B6F4-4F21-43E6-B12B-BD31792702E2%2FJulia%20LVA%20on%20ASE%20tutorial%7C7B52D37B-2A70-4829-B5E3-CAFBFE85B16F%2F%29
onenote:https://microsoft.sharepoint-df.com/teams/AzureStackLearnings/SiteAssets/AzureStack%20Learnings%20Notebook/Demo.one#Julia%20LVA%20on%20ASE%20tutorial&section-id={8264B6F4-4F21-43E6-B12B-BD31792702E2}&page-id={7B52D37B-2A70-4829-B5E3-CAFBFE85B16F}&end)
```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by emgerner-msft** [comment]
Rename anything with "julia" in it.

File: .gitignore [hunk 1]
```diff
@@ -105,8 +105,393 @@ venv.bak/
...
-.vscode
-
 # vi
 *~
+
+# julia binaries
+Research/julia/src/cloud-to-device-console-app/bin*
+Research/julia/src/cloud-to-device-console-app/obj/*
+Research/julia/src/ams-asset-player/bin*
+Research/julia/src/edge/modules/objectCounter/bin
```
Comment (emgerner-msft) [inline .gitignore:115]: How many of the changes in this file are manual changes to omit your specific files vs things you copied from something? I see a lot of cruft here with commit messages etc....

File: .vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (emgerner-msft) [inline .vscode/tasks.json:1]: a bunch of these vscode files look specific to your dev environment

File: .gitignore (hunk 1 above)
Comment (emgerner-msft) [reply .gitignore:115]: Beyond that, it looks like other projects have project-specific git ignores which you should probably model.

File: Research/julia/src/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+## Contents
+
+| File/folder       | Description                                |
+|----------------------|--------------------------------------------|
+| `src`                | Sample source code.                        |
+| `.gitignore`         | Defines what to ignore at commit time.     |
+| `README.md`          | This README file.                          |
+| `LICENSE`            | The license for the sample.                |
+| `SECURITY`           | Guidelines for reporting security issues   |
+| `CODE_OF_CONDUCT.md` | Open source code of conduct                |
```
Comment (emgerner-msft) [inline Research/julia/src/README.md:32]: I don't think you do or should have these files. I'm guessing you copy/pasted from somewhere? Remove, and check if you missed any other cleanup in this file.

**Comment by emgerner-msft** [comment]
At a high level a lot of these files look like copy/paste from other samples. At the very least we should document carefully from where, but if it's possible it might be better to script pulling them in from there or something like that. Otherwise as those services evolve and their samples are upgraded, ours will become slowly and subtly outdated.

File: .vscode/tasks.json (hunk 2 above)
Comment (julialieberman) [reply .vscode/tasks.json:1]: these vscode files are necessary to run LVA! (the tasks.json and launch.json files specifically)

File: .gitignore (hunk 1 above)
Comment (julialieberman) [reply .gitignore]: Committing those changes now!

File: .gitignore [hunk 4]
```diff
@@ -105,8 +105,5 @@ venv.bak/
 /.vs
 /staged-data-analytics
```
Comment (emgerner-msft) [inline .gitignore:107]: undo the changes to this file.

File: .vscode/tasks.json (hunk 2 above)
Comment (emgerner-msft) [reply .vscode/tasks.json:1]: why do they have to live in the root of the project is the main concern? they're specific to the lva sample so presumably that's where they should live

**Comment by emgerner-msft** [comment]
Related to the above comment on how much of this is copied from elsewhere, if you can point to what you've changed vs what is directly copied this will probably be a more productive review.
